### PR TITLE
feat(LDS-91): Track DVN Wire Usage + UX fix

### PR DESCRIPTION
**Pull request by syed-bcw**

### CODE REVIEW

**Review by amnbcw — APPROVED** [review]

File: src/utils/deploy.ts [hunk 1]
```diff
@@ -107,16 +108,24 @@ export class DeployUtils {
       : console.error(deploymentResponse.msg);
 
     if (deploymentResponse.isSuccess) {
+
+      const updateDVNConfig = await confirm({
+        message: 'Do you want to update DVN config?',
```
Comment (daniel-bcw) [inline src/utils/deploy.ts:113]: perhaps we can add `the` -> `Do you want to update the DVN config?`

File: src/utils/deploy.ts [hunk 2]
```diff
@@ -126,18 +135,9 @@ export class DeployUtils {
...
-            name: 'Yes',
-            value: true,
-          },
-          {
-            name: 'No',
-            value: false,
-          },
-        ],
+      const setupDVNs = await confirm({
+        message: 'Do you want to write DVNs on-chain?',
```
Comment (daniel-bcw) [inline src/utils/deploy.ts:139]: Curious, what does it actually mean `to write DVNs on -chain` ?

Comment (syed-bcw) [reply src/utils/deploy.ts:139]: had to write Wire instead of write

Comment (daniel-bcw) [reply src/utils/deploy.ts:139]: oh ok, lets change it then when u can

**Pull request merged**